src/components/Pricing/PricingPlan.tsx

```diff
@@ -1,209 +1,72 @@
-            data-button-type="cta"
-        >
-            {buttonLabel}
-        </a>
-    )
-
-    return (
-        <div className={`h-100 card ${className}`}>
-            <h2 className="mt-3 mb-1 tw-text-center tw-font-semibold">{name}</h2>
-            <div className="py-3 tw-text-center tw-flex tw-flex-col tw-items-center">
-                {button}
-                <div className="mt-4 mb-2 tw-pb-xxs tw-text-xl text-muted">{price}</div>
-                {planProperties}
-            </div>
-            <ol className="px-6 py-3 ml-0 list-group list-group-flush">
-                {!isFree ? (
-                    <li className="bg-transparent border-0 tw-px-0 tw-text-xl list-group-item">
-                        Everything in the Free tier, plus:
-                    </li>
-                ) : null}
-                {FEATURE_ORDER.map(feature => (
-                    <div key={FEATURE_INFO[feature].label}>
-                        <PricingPlanFeature
-                            info={FEATURE_INFO[feature]}
-                            value={features[feature]}
-                            tag="li"
-                            className="bg-transparent border-0 tw-px-0 tw-text-xl list-group-item"
+        <div className="tw-py-sm ml-0">
+            {isEnterprise && <div className="tw-text-xl tw-font-semibold tw-mb-sm">Everything in Business, plus:</div>}
+            {features.map(node => (
+                <div
+                    key={node.topic}
+                    className="tw-px-0 bg-transparent border-0 tw-text-xl list-group-itemtw-justify-between"
+                >
+                    <div className="tw-text-xl tw-font-semibold tw-flex tw-items-center">
+                        <CheckIcon
+                            className={`mr-2 icon-inline ${
+                                isEnterprise ? 'tw-text-violet-400' : 'tw-text-vermillion-300'
+                            } tw-inline`}
                         />
+                        <h5>{node.topic}</h5>
                     </div>
-                ))}
-            </ol>
+                    <ul className="tw-ml-2xl tw-mb-xs">
+                        {node?.features?.map(feature => (
+                            <div key={feature}>
+                                <PricingPlanFeature feature={SPOTLIGHT_FEATURE_INFO[feature]} tag="li" />
+                            </div>
+                        ))}
+                    </ul>
+                </div>
+            ))}
+            {/* {FEATURE_ORDER.map(feature => (
+                <div key={SPOTLIGHT_FEATURE_INFO[feature].label}>
+                    <PricingPlanFeature
+                        info={SPOTLIGHT_FEATURE_INFO[feature]}
+                        value={features[feature]}
+                        tag="li"
+                        className="tw-px-0 bg-transparent border-0 tw-text-xl list-group-item"
+                    />
+                </div>
+            ))} */}
         </div>
```

Comment:
Do we want to remove this or keep?

Reply:
😆 yes can remove the comment, left over from development hah